scripts/firedrake-install

```diff
@@ -1577,6 +1577,7 @@ if mode == "install" or not args.update_script:
                              "cmake",
                              "gfortran",
                              "git",
+                             "pkgconf", # for p4est
```

Comment:
The linter is complaining about this line

Reply:
Done.